client/login/magic-login/index.jsx

```diff
@@ -276,26 +280,30 @@ class MagicLogin extends Component {
 						onSendEmailLogin={ ( usernameOrEmail ) => this.setState( { usernameOrEmail } ) }
 						createAccountForNewUser
 					/>
-					<hr className="gravatar-magic-login__divider" />
-					<div className="gravatar-magic-login__footer">
-						<div>
-							{ translate(
-								'Do you prefer {{a}}logging in via password or Google/Apple instead{{/a}}?',
-								{
-									components: {
-										a: <a href={ loginUrl } />,
-									},
-								}
-							) }
-						</div>
-						<div>
-							{ translate( 'Any question? {{a}}Check our help docs{{/a}}.', {
-								components: {
-									a: <a href="https://gravatar.com/support" target="_blank" rel="noreferrer" />,
-								},
-							} ) }
-						</div>
-					</div>
+					{ ! isWPJobManager && (
```

Comment:
Maybe reverse this logic? What if we only show this for the pure Gravatar ID? (and remove for any other gravatar-powered client ID).